src/app/donation-start/donation-start.component.ts

```diff
@@ -821,6 +821,14 @@ export class DonationStartComponent implements AfterContentChecked, OnDestroy, O
           () => {
             this.analyticsService.logEvent('cancel', `Donor cancelled donation ${donation.donationId} to campaign ${this.campaignId}`),
             this.donationService.removeLocalDonation(donation);
+
+            // Removes match funds reserved timer if present
+            delete this.donation;
+
+            // Go back to previous step to encourage donor to try again
+            this.stepper.previous();
```

Comment:
We don't know what step the donor is on when this asynchronous process completes. What about `reset()`?